Makefile

```diff
@@ -26,26 +25,18 @@ all: build/omni_lock build/always_success
 all-via-docker: ${PROTOCOL_HEADER}
 	docker run --rm -v `pwd`:/code ${BUILDER_DOCKER} bash -c "cd /code && make"
 
-
 build/always_success: c/always_success.c
 	$(CC) $(OMNI_LOCK_CFLAGS) $(LDFLAGS) -o $@ $<
 	$(OBJCOPY) --only-keep-debug $@ $@.debug
 	$(OBJCOPY) --strip-debug --strip-all $@
 
 build/secp256k1_data_info_20210801.h: build/dump_secp256k1_data_20210801
 	$<
+	rm -f deps/secp256k1-20210801/src/precomputed_ecmult.h
 
-build/dump_secp256k1_data_20210801: c/dump_secp256k1_data_20210801.c $(SECP256K1_SRC_20210801)
+build/dump_secp256k1_data_20210801: c/dump_secp256k1_data_20210801.c
 	mkdir -p build
-	gcc -I deps/secp256k1-20210801/src -I deps/secp256k1-20210801 -o $@ $<
-
-
-$(SECP256K1_SRC_20210801):
-	cd deps/secp256k1-20210801 && \
-		./autogen.sh && \
-		CC=$(CC) LD=$(LD) ./configure --with-bignum=no --enable-ecmult-static-precomputation --enable-endomorphism --enable-module-recovery --host=$(TARGET) && \
-		make src/ecmult_static_pre_context.h src/ecmult_static_context.h
-
+	gcc -I deps/secp256k1-20210801/src -o $@ $<
```

Comment:
Why are these line removed? New secp256k1 lib don't require config?

Reply:
[precomputed_ecmult.c](https://github.com/bitcoin-core/secp256k1/blob/0cdc758a56360bf58a851fe91085a327ec97685a/src/precomputed_ecmult.c) already exists in the repositorie and does not need to be regenerated.

Reply:
it is too large, about 2.3M bytes. We need a smaller one.

Reply:
But the exported secp256k1_data_20210801 is still 1M (1048576).
The size still needs to be controlled at around 20k? (ckb-auth)

Reply:
it's ok to have same size as before (1M).